pbnh/tests/paste_curl_test.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python3
+import pycurl
+import unittest
+import json
+import hashlib
+import tempfile
+import os
+
+from pbnh import conf
+from pbnh import app
+
+config = conf.get_config().get('server')
+host = config.get('bind_ip')
+port = config.get('bind_port')
+URL = 'http://{}:{}/'.format(host, port)
+
+
+class TestPost(unittest.TestCase):
+    def setUp(self):
+        self.c = pycurl.Curl()
+        self.c.setopt(pycurl.URL, URL)
+        self.db_fd, app.app.config['DATABASE'] = tempfile.mkstemp()
```

Comment:
Ahh so creating a new db in the setUp means that the curl command will use that DB? Didn't figure that out. I'll edit it to do that